### DISCUSSION TITLE
Printing posters on multiple smaller papers

### DISCUSSION THREAD

**Discussion by jpyllman** [sumatrapdfreader/sumatrapdf#4313] [Ideas]
Hi,

Would be nice to be able to print PDF as poster. That meaning instead of fitting one page by scaling it down you would like to print it by scaling up and use multiple pages of the smaller paper size.

**Comment by GitHubRulesOK** [comment]
Whilst poster decimation is a MuPDF (Mutool) function it is very basic (just integers) so split a page into whole parts eg 2 x 2 then they can be printed as such.

For full control of printer dialog for setting overlap margins etc you need a print orientated editor like Adobe include in their Lighter pro products such as "Reader DC" One of the many hidden limited editing functions that make it slower to run.

For command line nup/poster there are few products available such as PDFCPU and CPDF manipulation but "Poster"  can be exceptionally hard to emulate as IT requires steps like object trimming and PDF is adverse to such vector splitting without a slow and ponderous edit.

A lighter alternative to Acrobat is my 3rd most favorite PDF tool Tracker X-Change editor where the Editing abilities are more obvious
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c52a2c7e-342d-4bd4-9475-bcce3c930fee)

**Reply by jpyllman** [reply]
OK. It would be a nice thing, but not important. I probably would use it 3 times in my life.

**Comment by jpyllman** [comment]
krop on linux is also an option I think.
